_episodes/05-starting-jekyll.md

```diff
@@ -21,4 +21,57 @@ keypoints:
 - use it again in another page
 - change the value of the global parameter and study the effect
 
+
+> ## Creating your own variable
+>
+> Let's practice making and using variables.
+> Think of a variable you may want to use in your `index.md` page.
+> Some possible options include: your name, the institution you work at, the PI of your lab group.
+> Create the variable and reference it in your `index.md` and save/commit that to your repository.
+> What did you add to your `index.md` to create this variable?
+> Where did you add the line above to your `index.md`?
+> How did you reference that variable?
+>
+> > ## Solution
+> >
+> > We will create the `pi` variable by adding the following line to the yaml header.
+> > ~~~
+> > pi: "Ada Lovelace"
+> > ~~~
+> >
+> > Here is an example of using it in the `index.md` file
+> > ~~~
+> > This project is lead by {{ pi }}.
+> > ~~~
+> > {: .output}
+> {: .solution}
+{: .challenge}
+
+
+> ## Making a global variable
+>
+> Grad student Davis wants to use their current institution name throughout their personal website
+> but will soon be leaving to become a postdoc at another institution.
+> They have created a variable `current_institution` in the YAML header of the `index.md` pages on their site.
+> What changes would they make to use this variable on other pages as well?
+> How might they need to change the following text on their `index.md` page after those changes?
+>
+> ~~~
+> Davis currently works at {{ current_institution }}.
+> ~~~
+> {: .source}
+>
+> > ## Solution
+> >
+> > They need to move the variable to the `_config.yml`
+> > Then they would change the index code to the following.
+> > ~~~
+> > Davis currently works at {{ site.current_institution }}.
```

Comment:
This one is nice! We could also have another where we try to refer to a `{{ page.(...) }}` variable that is defined in another document.

Reply:
I'd be +1 to adding an optional exercise about this, but don't think it's crucial to include it as a core exercise.

Reply:
I think that would make a great optional exercise.  Could we decouple it from this PR and make a new issue for it?  I'm thinking it would be good to get this PR merged before we work on the lesson content in next sprint.